stestr/tests/test_return_codes.py

```diff
@@ -73,6 +73,7 @@ def assertRunExit(self, cmd, expected, subunit=False, stdin=None):
             self.assertEqual(
                 p.returncode, expected,
                 "Stdout: %s; Stderr: %s" % (out, err))
+            return (out, err)
```

Comment:
Part of me feels like this is not a good way to do this. I always think an assert function shouldn't have a return at all. It either works and does nothing or fails and explodes. But I guess this is fine for now, it's probably better than calling subprocess by hand 2 or 3 times in the test function and doing the asserts manually. Maybe as a follow up change we can extract the subprocess piece into a separate private function and have the assertMethod call that for subprocessing and we can remove the return here. Because we can use that new method in the test directly.